src/examples/tags/TagsShape.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Span } from '@zendeskgarden/react-typography';
+import { Tag } from '@zendeskgarden/react-tags';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => (
+  <Row>
+    <Col textAlign="center">
+      <Tag>
+        <Span>Algae</Span>
+      </Tag>
+    </Col>
+    <Col textAlign="center">
+      <Tag isPill>
+        <Span>Moss</Span>
+      </Tag>
+    </Col>
+    <Col textAlign="center">
+      <Tag isRound>
+        <Span>12</Span>
```

Comment:
I think the team has allowed a max of two characters, but one looks better. Maybe only show a single digit?